tests/test_oauth_backend.py

```diff
@@ -0,0 +1,39 @@
+from tests import SiteTest, manager
+
+class TestOauthBackend(SiteTest):
+    """ Test cases for the oauth.py file """
+
+    def test_get(self):
+        """ Make sure the get function returns nothing """
+        self.assertIs(manager.oauth_backend.get(), None)
+
+    def test_delete(self):
+        """ Make sure the delete function returns nothing """
+        self.assertIs(manager.oauth_backend.delete(None), None)
+
+    def test_logout(self):
+        """ Make sure at least apart of logout is working :/ """
+        self.assertIs(manager.oauth_backend.logout(), None)
```

Comment:
these would probably all be cleaner with the `assertIsNone` as shown above

Reply:
triple done 👍